appservice/src/github/IConnectToGitHubWizardContext.ts

```diff
@@ -12,5 +12,5 @@ export interface IConnectToGitHubWizardContext extends IActionContext {
     repoData?: gitHubRepoData;
     branchData?: gitHubBranchData;
     client?: SiteClient;
-    node?: AzureTreeItem;
+    subscriptionContext?: ISubscriptionContext;
```

Comment:
Rather than having a context being a property on the `IConnectToGitHubWizardContext`, you should be able to just have it extend `ISubscriptionContext` as well.  Then whenever you you need `IConnectToGitHubWizardContext`, you can just do `Object.assign()` like this https://github.com/microsoft/vscode-azurefunctions/blob/51f4d50e978d292e7d46579c918eb229b7774178/src/tree/SubscriptionTreeItem.ts#L82-L87

Reply:
Or, just leave them as two separate entities.  For instance, you set `context.subscriptionContext = subscriptionContext` but you could also just pass in the subscriptionContext to `getGitHubJsonResponse` as well.

I wouldn't worry too much about it, I think with the new GitHub auth experience coming into VS Code stable, I'll have to come back and refactor a lot of this code so do whatever's simpler for you.

Reply:
I ended up having `IConnectToGithubWizardContext` extend `ISubscriptionContext`  👍